From 2a4e324fece7b2cdd36f106b38973e79d6dfae7f Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Fri, 23 Jun 2017 09:49:05 +0800
Subject: [PATCH 1993/5242] MLK-15143 ARM64: dts: freescale: imx8mq: increase
 CMA size

commit  8ca553b79c62e84e943919a2adf137a3930358d9 from
https://source.codeaurora.org/external/imx/linux-imx.git

Increase CMA size to 640MB for i.MX8MQ.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index 847d1a7..55f6f65 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -52,8 +52,8 @@
 		linux,cma {
 			compatible = "shared-dma-pool";
 			reusable;
-			size = <0 0x04000000>;
-			alloc-ranges = <0 0x40000000 0 0x20000000>;
+			size = <0 0x28000000>;
+			alloc-ranges = <0 0x40000000 0 0x80000000>;
 			linux,cma-default;
 		};
 	};
-- 
1.7.9.5

